app/controllers/uploads_controller.rb

```diff
@@ -274,4 +351,8 @@ def send_file_local_upload(upload)
     send_file(file_path, opts)
   end
 
+  def upload_failed(message)
+    message = message.is_a? Array ? message : [message]
+    render json: failed_json.merge(errors: message), status: 422
```

Comment:
Any reason why you couldn't use `render_json_error(message)` here?

Reply:
The reason is because I forgot that existed, will fix :)

Reply:
Oh also I don't even use this method anyway! Will just get rid of it